packages/tentales/src/service/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { ServiceCaller, Services, Service, ReduceObject } from "../types"
+
+export function convertServiceMethodsToServices(
+  serviceMethods: ServiceCaller[],
+): Services {
+  return serviceMethods.reduce(
+    (acc, method): Service => {
+      acc[method.name] = method.func
```

Comment:
Same here, maybe do `return { ...acc, [method.name]: method.func }` to keep it immutable?